### PR TITLE
Send `last_edited_by_editor_id` to Publishing API

### PR DESCRIPTION
**Pull request by pezholio**
This is a new field that will allow us to identify the last person who has edited an Edition within the Publishing API.

### CODE REVIEW

File: test/support/presenter_test_helpers.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module PresenterTestHelpers
+  def it_includes_last_edited_by_editor_id
+    should "[:last_edited_by_editor_id]" do
+      editor = FactoryBot.create(:user)
+      edition.actions.create!(request_type: Action::CREATE, requester: editor)
+
+      assert_equal editor.uid, result[:last_edited_by_editor_id]
+    end
+  end
+end
```
Comment (baisa) [inline test/support/presenter_test_helpers.rb:10]: We should include test that ensures us we are actually sending last editor id, not id of the user who created the edition. Although it seems like we do not have access to this information atm.

Comment (pezholio) [reply test/support/presenter_test_helpers.rb:21]: Cool, I've updated the tests to cover both eventualities. Looking at the behaviour of the code, the only time a `CREATE` action is created is when creating a brand new edition (i.e one without any children), if a new edition is created from an existing edition, the creates a `NEW_VERSION` action.

File: test/support/presenter_test_helpers.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    test "[:last_edited_by_editor_id] should return the creator of a new version" do
+      editor = FactoryBot.create(:user)
+      edition.actions.create!(request_type: Action::NEW_VERSION, requester: editor)
+
+      assert_equal editor.uid, result[:last_edited_by_editor_id]
+    end
+
+    test "[:last_edited_by_editor_id] should return nil when there are no new version or create actions" do
+      assert_equal nil, result[:last_edited_by_editor_id]
```
Comment (baisa) [inline test/support/presenter_test_helpers.rb:18]: Consider using `assert_nil` as `asset_equal nil` will soon be deprecated.

File: app/presenters/formats/edition_format_presenter.rb [hunk 3]
```diff
@@ -97,5 +98,9 @@ def major_change?
     def public_updated_at
       edition.public_updated_at || edition.updated_at
     end
+
+    def last_edited_by_editor_id
+      edition.created_by&.uid
+    end
```
Comment (baisa) [inline app/presenters/formats/edition_format_presenter.rb:104]: Maybe worth adding a comment here to explain that we do not really have access to information who edited the edition last, so we rely on created_by.

**Review by baisa — COMMENTED** [review]
Once we add explanatory comment (as below) it LGTM

**Review by baisa — APPROVED** [review]

**Pull request merged**